src/systems/physics/Physics.hh

```diff
@@ -51,6 +51,7 @@
 
 #include <gz/sim/config.hh>
 #include <gz/sim/System.hh>
+#include <gz/sim/Link.hh>
```

Comment:
can you move this include to `Physics.cc` given that it's only used there? and minor style nit: put it in alphabetical order with double quotes.